TileFormats/Batched3DModel/README.md

```diff
@@ -119,16 +120,20 @@ The vertex shader can be modified at runtime to use `a_batchId` to access indivi
 
 When a Batch Table is present or the `BATCH_LENGTH` property is greater than `0`, the `batchId` attribute (with the parameter semantic `_BATCHID`) is required; otherwise, it is not.
 
-Although not strictly required, clients may find the glTF [CESIUM_RTC](https://github.com/KhronosGroup/glTF/tree/master/extensions/1.0/Vendor/CESIUM_RTC) extension useful for high-precision rendering.
+### Coordinate Reference System (CRS)
 
-## File Extension
+Vertex positions are defined according to a right-handed coordinate system where Y-axis is up. [glTF Coordinate System and Units](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#coordinate-system-and-units)
 
-`.b3dm`
+Vertex positions may be defined relative-to-center for high-precision rendering [1]. If defined, `RTC_CENTER` specifies the center position and all vertex positions are treated as relative to this value.
```

Comment:
Here it should be noted that `RTC_CENTER` is z-up whereas vertex positions are y-up.

In general I think our coordinate system messaging is still not _quite_ there... still need to think about the clearest way possible to describe it.